views/layouts/main.php

```diff
@@ -28,111 +28,121 @@
 
 <?php $this->beginBody() ?>
     <div class="wrap">
-        <?php
-        NavBar::begin([
-            'brandLabel' => '<span class="yii-logo"></span> YiiPowered',
-            'brandUrl' => Yii::$app->homeUrl,
-            'options' => [
-                'class' => 'navbar-inverse navbar-fixed-top',
-            ],
-        ]);
-
-        $menuItems = [
-            [
-                'label' => Html::tag('span', '<span> ' . Yii::t('app', 'Explore projects'). '</span>', ['class' => 'fa fa-search']),
-                'encode' => false,
-                'url' => ['/projects'],
-                'linkOptions' => ['alt' => Yii::t('app', 'Explore projects'), 'title' => Yii::t('app', 'Explore projects')],
-            ],
-            [
-                'label' => Html::tag('span', '<span> ' . Yii::t('app', 'RSS feed') . '</span>', ['class' => 'fa fa-rss-square']),
-                'encode' => false,
-                'url' => ['/project/rss'],
-                'linkOptions' => [
-                    'alt' => Yii::t('app', 'RSS feed'),
-                    'title' => Yii::t('app', 'RSS feed'),
-                ]
-            ],
-            [
-                'label' => Yii::t('project', 'Top {n}', ['n' => Yii::$app->params['project.maxTopProjects']]),
-                'url' => ['/project/top-projects'],
-            ]
-        ];
-        if (Yii::$app->user->isGuest) {
-            $menuItems[] = [
-                'label' => Html::tag('span', '<span> ' . Yii::t('app', 'Login') . '</span>', ['class' => 'fa fa-sign-in']),
-                'encode' => false,
-                'url' => ['/site/login'],
-                'linkOptions' => ['alt' => Yii::t('app', 'Login'), 'title' => Yii::t('app', 'Login')],
-            ];
-        } else {
-            $menuItems[] = [
-                'label' => Yii::t('user', 'Manage users'),
-                'url' => ['/user/index'],
-                'visible'=> \Yii::$app->user->can('manage_users'),
-            ];
-            $menuItems[] = [
-                'label' => Html::tag('span', '<span> ' . Yii::t('app', 'Bookmarks') . '</span>', ['class' => 'fa fa-bookmark']),
-                'encode' => false,
-                'url' => ['/project/bookmarks'],
-                'linkOptions' => ['alt' => Yii::t('app', 'Bookmarks'), 'title' => Yii::t('app', 'Bookmarks')],
-            ];
-            $menuItems[] = ['label' => Yii::$app->user->identity->username, 'url' => ['/user/view', 'id' => \Yii::$app->user->id]];
-            $menuItems[] = [
-                'label' => Html::tag('span', '<span> ' . Yii::t('app', 'Logout') . '</span>', ['class' => 'fa fa-sign-out']),
-                'url' => ['/site/logout'],
-                'encode' => false,
-                'linkOptions' => [
-                    'data-method' => 'post',
-                    'alt' => Yii::t('app', 'Logout'),
-                    'title' => Yii::t('app', 'Logout'),
+        <div class="content">
```

Comment:
What's this change for? What behavior does it fix?

Reply:
It's for footer-fix. Footer places at the bottom by "display: flex".
div.content tag need for flex elements.